docs/api/event-brokers.rst

```diff
@@ -231,3 +235,51 @@ according to the security protocol being used. The following implementation show
 
     for message in consumer:
         print(message.value)
+
+SQL Event Broker
+-----------------
+
+It is possible to use  an SQL database as an event broker. Connections to databases are established using
+`SQLAlchemy <https://www.sqlalchemy.org/>`_, a Python library which can interact with many
+different types of SQL databases, such as `SQLite <https://sqlite.org/index.html>`_,
+`PostgreSQL <https://www.postgresql.org/>`_ and more. The default Rasa installation allows connections to SQLite
+and PostgreSQL databases, to see other options, please see the
+`SQLAlchemy documentation on SQL dialects <https://docs.sqlalchemy.org/en/13/dialects/index.html>`_.
+
+
+Adding a SQL Event Broker Using the Endpoint Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can use an endpoint configuration file to instruct Rasa to stream
+all events to your SQL event broker. To do so, add a ``event_broker`` section to your
+endpoint configuration, e.g. ``endpoints.yml``. For example, a valid SQLite configuration
+could look like the following:
+
+.. code-block:: yaml
+
+    event_broker:
+      type: SQL
+      dialect: sqlite
+      db: events.db
+
+PostgreSQL databases can be used as well:
+
+.. code-block:: yaml
+
+    event_broker:
+      type: SQL
+      host: 127.0.0.1
+      port: 5432
+      dialect: postgresql
+      username: myuser
+      password: mypassword
+      db: mydatabse
+
+Then, instruct Rasa to use the endpoint configuration and SQL producer by adding
```

Comment:
Actually we can skip that. `rasa run` does exactly the same (looks for `endpoints.yml` by default).
Maybe it makes sense to add a short general section on the top, that all event brokers are configured in the endpoints file and that rasa looks by default for a `endpoints.yml` and that they have to specify other values explicitly with `--endpoints`.